dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/FATSuite.java

```diff
@@ -18,26 +18,7 @@
 @RunWith(Suite.class)
 @MinimumJavaLevel(javaLevel = 11)
 @SuiteClasses({
-                JaxRsIntegration.class,
                 JaxRsIntegrationWithConcurrency.class,
```

Comment:
Where did all the other tests go?

Reply:
Oops. That change is Left over from debugging.